internal/target/http.go

```diff
@@ -124,14 +188,17 @@ func (target *Target) ProcessTarget(payload interface{}) (int, error) {
 		Code:   resp.StatusCode,
 		Body:   body,
```

Comment:
@anubhavbaweja21 Can you please attach only starting 250bytes of body, instead of whole to this? This will help us log the response body too, as well as maintain in db.